code/modules/events/spider_terror.dm

```diff
@@ -22,23 +22,31 @@
 			if(temp_vent.parent.other_atmosmch.len > 50)
 				vents += temp_vent
 	var/spider_type
-	var/infestation_type = pick(1, 2, 3, 4, 5)
+	var/highpop_trigger = 80
+	var/infestation_type = pick(1, 2, 3, 4)
```

Comment:
best to put the pick in the else of the if. Slight performance stuff but mostly for consistency